bin/pygrb/pycbc_grb_trig_cluster

```diff
@@ -187,10 +185,10 @@ with h5py.File(args.trig_file, "r") as h5f:
 # empty file (no triggers), so just copy the file
 if not time.size:
     shutil.copyfile(args.trig_file, outfile)
+    err_msg = "trigger file is empty\ncopied input file to {}".format(outfile)
     if args.verbose:
-        print("trigger file is empty")
-        print("copied input file to {}".format(outfile))
-    sys.exit(0)
+        print(err_msg)
+    raise RuntimeError(err_msg)
```

Comment:
You can avoid printing a message if you are going to raise an exception with the same message right after.